website_blog_excerpt_img/templates/blog.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2019 Tecnativa - Jairo Llopis
+     License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html). -->
+
+<odoo>
+
+    <template
+        active="True"
+        customize_show="True"
+        id="seo_image_floating"
+        inherit_id="website_blog.blog_post_short"
+        name="Floating social share image"
+        priority="50"
+    >
+
+        <xpath expr="//*[hasclass('o_blog_post_teaser')]/.." position="replace">
+            <div class="row">
+                <!-- Wrap text content with a .row element -->
+                <div class="col">$0</div>
+
+                <!-- Insert social share thumbnail next to teaser -->
+                <div class="col-5 text-center">
```

Comment:
Here uses text-center, but 'excerpt-img' has the "float:right;" style...

Reply:
Good catch, thanks. Fixed.